src/dialog/overflow-menu/overflow-menu.stories.ts

```diff
@@ -74,7 +74,7 @@ storiesOf("Components|Overflow Menu", module)
 					<ibm-overflow-menu-option href="https://www.ibm.com" (selected)="selected($event)" (click)="click($event)">
 						An example option that is really long to show what should be done to handle long text
 					</ibm-overflow-menu-option>
-					<ibm-overflow-menu-option href="https://www.ibm.com" (selected)="selected($event)">Option 2</ibm-overflow-menu-option>
+					<ibm-overflow-menu-option href="https://www.ibm.com" target="_blank" (selected)="selected($event)">Option 2</ibm-overflow-menu-option>
```

Comment:
```suggestion
					<ibm-overflow-menu-option href="https://www.ibm.com" target="Target.blank" (selected)="selected($event)">Option 2</ibm-overflow-menu-option>
```

Reply:
Would need to ...

```
import {Target} from 'carbon-components-angular';
```

Reply:
I don't dislike this idea at all. It's better than dealing with strings and makes the validation kind of redundant.
Let's see what the other folks say about it.

Reply:
I like the idea of using enums to set the value of the inputs!

Reply:
This works for sure, so long as it doesn't require the `enum` to be added as a property to the component ... I know that was an issue in the past, and it makes usage very awkward

Reply:
Templates never do type validation, unfortunately, so, strings can be used as well. But if importing the `Targets` enum is an issue we may well stick to strings? Or should we keep both options?

Reply:
We can keep both options for sure ... I'm not so worried about template type checking as I am the compiler requiring the enum to be assigned to a local property in order to be picked up, that's historically been the issue with enums in templates